### PR TITLE
PEP8 fixes

### PR DESCRIPTION
**Pull request by e-kwsm**
#### Describe Bug or Feature

The python codes do not comply with some of PEP8.

#### Describe Your Change

Fix the followings:

- [E101](https://github.com/PyCQA/pycodestyle/blob/2.9.1/docs/intro.rst#error-codes)
- [E711](https://github.com/PyCQA/pycodestyle/blob/2.9.1/docs/intro.rst#error-codes)
- [E712](https://github.com/PyCQA/pycodestyle/blob/2.9.1/docs/intro.rst#error-codes)
- [E722](https://github.com/PyCQA/pycodestyle/blob/2.9.1/docs/intro.rst#error-codes)
- [F632](https://github.com/PyCQA/pycodestyle/blob/2.9.1/docs/intro.rst#error-codes)
- [W605](https://github.com/PyCQA/pycodestyle/blob/2.9.1/docs/intro.rst#error-codes)

#### Link to Design Doc

https://openpbs.atlassian.net/wiki/spaces/DG/pages/1481670667/OpenPBS+Coding+Standards#OpenPBSCodingStandards-PythonCodingStyle

### CODE REVIEW

**Comment by e-kwsm** [comment]
I wonder why `checkpep8` has passed…
In this PR I don’t address E501, i.e. line length, since it requires a bunch of changes.

https://dev.azure.com/subhas0137/subhas/_build/results?buildId=1305&view=logs&j=b133c148-c6df-5fdc-3302-63cb364d6c67&t=101078ac-3490-51ec-8d4d-92e465b411b3&l=81

**Review by nithinj — APPROVED** [review]
LGTM!

Can you rebase and push the changes. I need to see why the CI is failing. Thanks!

File: test/tests/functional/pbs_node_rampdown.py [hunk 1]
```diff
@@ -6420,7 +6420,7 @@ def test_release_job_array(self):
         # Below variable is being used for the accounting log match
         # which is currently blocked on PTL bug PP-596.
         # new_exec_host_esc = self.job1_exec_host.replace(
-        # "*", "\*").replace("[", "\[").replace("]", "\]").replace("+", "\+")
+        # "*", r"\*").replace("[", r"\[").replace("]", r"\]").replace("+", r"\+")
```
Comment (nithinj) [inline test/tests/functional/pbs_node_rampdown.py:6423]: pep8 check is failing on this:
```
E501 line too long (81 > 79 characters)
        # "*", r"\*").replace("[", r"\[").replace("]", r"\]").replace("+", r"\+")
                                                                             ^
```
you can see the failure for yourself on azure pipelines. please fix the CI before we proceed to merge this.

**Review by nithinj — CHANGES_REQUESTED** [review]

File: test/tests/functional/pbs_node_rampdown.py (hunk 1 above)
Comment (e-kwsm) [reply test/tests/functional/pbs_node_rampdown.py:6423]: There are a bunch of too-long (E501) lines, and it is rather a work of formatter, yapf or black.

Comment (nithinj) [reply test/tests/functional/pbs_node_rampdown.py:6423]: Can you push the changes once it is done?

Comment (e-kwsm) [reply test/tests/functional/pbs_node_rampdown.py]: Done.

**Review by bayucan — APPROVED** [review]
Looks good.

**Pull request merged**